topics/appendix/tools/intellij_platform_gradle_plugin/tools_intellij_platform_gradle_plugin.md

```diff
@@ -104,17 +118,11 @@ Then invoke the <control>Reload All Gradle Projects</control> action from the <c
 
 </tab>
 
-<tab title="Earlier versions">
-
-No additional IDE settings are required.
-
-</tab>
-
 </tabs>
 
-The attaching sources is handed by the Plugin DevKit plugin thus it's recommended to use always the latest available IDE release.
+The attaching sources operation in the IDE is handled by the Plugin DevKit plugin thus it's always recommended to use the latest available IDE release.
```

Comment:
I found this a bit unclear. 

The DevKit plugin handled the attaching sources so it is recommended to use the latest plugin version available or the latest IDE release?